### PR TITLE
Remove google plus

### PR DESCRIPTION
**Pull request by therealjuan**
1. Pull the branch or run the demo service
2. Check on every page there is no Google Plus in the footer
3. #winning

### CODE REVIEW

**Comment by nottrobin** [comment]
In small screen view, these icons now appear on the right:

image

Where they used to appear more left-aligned (though still not perfect):

image

I would guess the ideal would be to match snapcraft.io:

image

**Comment by bartaz** [comment]
I guess the same will be true for snapcraft.io, which was also updated (but not deployed yet).

**Comment by therealjuan** [comment]
**Confirmation from a designer:**
The solution is to remove the right alignment in text as the mentioned style with @lyubomir-popov during standup this morning was to leave the gap at the end of the four columns to make sure the text is aligned on the same spot when you have a col-4 on the right hand side as is this case leaving the empty gap of the fourth icon in the footer.

**Comment by lyubomir-popov** [comment]
@bartaz @nottrobin My suggestion for the icons would be:

mobile: 
image


tablet:
image

desktop:
image


On the p-inline-list:
<ul class="p-inline-list--compact u-align--right u-no-padding--left u-no-margin--left u-no-margin--bottom">

(Requires a bit of reshuffle of the markup and some moble classes:)
image


By the way is the global nav supposed to end up at the bottom? looks very strange:
image

**Comment by lyubomir-popov** [comment]
@therealjuan only saw your reply after I posted, but I agree - on desktop it can be left aligned in a 4-col container. Either way is fine with me.

File: static/sass/_patterns_social-icons.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
     background-image: url('#{$assets-path}6bfc3d6a-build.snapcraft.social.optimize.svg');
     background-repeat: no-repeat;
     background-size: 160px 80px; // 2x4 icons 40x40px each (in 2 rows for regular and hover state)
+    color: transparent;
     display: inline-block;
     height: 40px;
```
Comment (lyubomir-popov) [inline static/sass/_patterns_social-icons.scss]: I'm surprised to see all dimensions hardcoded as px values. This is a problematic practice, especially when we are planning a bump in font-size on bigger screens. All our patterns use rem values which will resolve to different px values once a rem value is changed on a breakpoint.

**Comment by therealjuan** [comment]
Going through the changes proposed by @lyubomir-popov and updating the list of links in the bottom

**Comment by therealjuan** [comment]
Right now, there was a problem with the list of links at the very bottom in the footer that were acting weird. If we are happy with the below, I'll do the proposed changes in snapcraft.io and build:

**Desktop**
image

**Tablet**
image

**Mobile**
image

**Comment by lyubomir-popov** [comment]
@therealjuan @kwm14 I've filed a bug for the p-inline-list--middot: https://github.com/vanilla-framework/vanilla-framework/issues/2147

I think it's ok for now to do it the way you have demonstrated. That pattern will need to be rewritten to use flex instead of inline to behave as you'd expect, i.e. skip dot and gap on wrapping.

here's the space-between utility I use in maas make items like the back to top / social icons use all available space before wrapping:

```
@mixin patterns_space-between {
  // Allow items to stay on the same line until the space between them vanishes
  // advantage over floats: when the width of the parent is less than the combined width of the children, the children stack, rather than force their contents to wrap.
  // advantage over columns: wrapping only occurs when elements actually run into each other, rather than at a predefined breakpoint
  // N.B.: Do not apply on the same level as elements having ::before or ::after, as it will factor them into the space-between calculation

  .p-space-between {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;

    @media (max-width: $breakpoint-small) {
      flex-direction: column;
    }

    //utility to align things to the right in a container with flex-direction: column
    .p-space-between__align-right {
      @media (max-width: $breakpoint-small) {
        align-self: flex-end;
      }
    }
  }
}
```

**Comment by nottrobin** [comment]
I completely agree with @bartaz, we should get this merged as quickly as we can, without going down other avenues.

My comment was only that the new mobile layout of the icons seemed potentially worse than the original (a regression). But it wasn't terrible, so if we know we're going to  be evolving the footer anyway soon, maybe we should just merge @therealjuan's initial implementation ASAP, and continue evolving the footer in a new PR?

**Comment by therealjuan** [comment]
Just added the initial implementation as mentioned by @nottrobin so we can take care of footer updates in a follow-up PR.

**Pull request merged**